benchmarks/metrics/coco/mean_average_precision_bucket_performance.py

```diff
@@ -23,9 +23,9 @@ def produce_random_data(include_confidence=False, num_images=128, num_classes=20
     images = []
     for _ in range(num_images):
         num_boxes = math.floor(25 * random.uniform(0, 1))
-        classes = np.floor(np.random.rand(num_boxes, 1) * num_classes)
+        classes_in_image = np.floor(np.random.rand(num_boxes, 1) * classes)
```

Comment:
This is the only place where a meaningful rename of an existing variable had to occur. LMK what you think.

Reply:
(Also in mean_average_precision_performance.py)

Reply:
(Also in recall_performance.py)